### DISCUSSION TITLE
1/6 Tall segments

### DISCUSSION THREAD

**Discussion by sjamesmccarthy** [rxhanson/Rectangle#474] [Ideas]
Feature suggestion.

You have 1/6 Top|Bottom & Left|Center|Right (this almost feels redundant with Top Left, Top Right, Bottom Left, Bottom Right)
1|2|3
4|5|6

but what about full vertical slices for 1/6 like you have fro 1/4, 1/2 and 1/3
1|2|3|4|5|6

People with ultra-wide displays may find this option useful. Example, I would use the vertical 6ths with apps like Messages, Reminders, Notes, Music (when minimized to compact), Home app, maybe even Safari (not entirely sold on that one), but also any iOS app being opened on an M1 Mac may find that size useful too. 

Thanks.

**Comment by rxhanson** [comment]
Thanks for the suggestion. Here's a post I wrote a while back about why this isn't in Rectangle: https://medium.com/ryan-hanson/custom-window-dimension-shortcuts-in-macos-e7df347dfd24

The 2x3 grid was asked for much more than the 1x6 grid.

Even though this can be accomplished with the custom shortcut recorder in my paid superset of Rectangle, [Hookshot](https://hookshot.app), my plan is to add it in to Hookshot as regular window actions as well - just haven't gotten around to it yet.

If someone really wants to add it to Rectangle, I'd still consider a PR but I'd only look to merge it if it's done elegantly in the UI somehow.
